build-support/travis/travis.yml.mustache

```diff
@@ -471,44 +524,23 @@ linux_rust_tests: &linux_rust_tests
   sudo: required
   language: python
   python: *python3_version
-  stage: *test
-  before_script:
-    - ulimit -c unlimited
-    - ulimit -n 8192
   script:
     - ./build-support/bin/travis-ci.sh -be
 
-base_osx_rust_tests: &base_osx_rust_tests
+osx_rust_tests: &osx_rust_tests
```

Comment:
This should use the `native_engine_cache_config` but currently doesn't. Maybe let's hoist that up into `base_rust_tests` if yaml will let us do that?